nimbleclient/v1/field.py

```diff
@@ -0,0 +1,85 @@
+#
```

Comment:
I think we should rename this from `field.py` to `search.py` or `query.py`.

Reply:
Lets' rename to `searchFilter.py` ?